### ISSUE TITLE
Regarding the extraction rules for "result media images" and the question of why there are many images in the Markdown content captured but few images extracted

### ISSUE THREAD

**Issue by lexang** [unclecode/crawl4ai#297]
Regarding the extraction rules for "result media images" and the question of why there are many images in the Markdown content captured but few images extracted

**Comment by unclecode** [comment]
@lexang I'm not very clear about your question here. Maybe you can share your code; over there, we can discuss. Just let you know that there are a bunch of parameters that you can have a kind of very nice control with high granular control over the type of images you want to be in the markdown. Let me know more about your case, and I can help you.

**Comment by aravindkarnam** [comment]
@lexang I believe this [explanation](https://github.com/unclecode/crawl4ai/discussions/488#discussioncomment-11882171) should answer your question. 

TLDR;
The extracted images go through a filter to toss out very small images (like the ones on buttons, links, icons, logos etc that's meant to just enhance design). You can adjust the threshold on this filter to get all images as well.
